### PR TITLE
fix: Make binary installs consistent

### PR DESCRIPTION
**Pull request by SuperQ**
Update the blackbox_exporter and snmp_exporter installs to be consistent with other exporter package installs.

Fixes: https://github.com/prometheus-community/ansible/issues/96

### CODE REVIEW

File: roles/blackbox_exporter/tasks/install.yml [hunk 1]
```diff
@@ -15,33 +15,56 @@
...
-    creates: "/tmp/blackbox_exporter-{{ blackbox_exporter_version }}.linux-{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}/\
-              blackbox_exporter"
-  register: _download_binary
-  until: _download_binary is succeeded
-  retries: 5
-  delay: 2
-  delegate_to: localhost
-  check_mode: false
-  when: not blackbox_exporter_skip_install
+- name: Discover latest version
```
Comment (gardar) [inline roles/blackbox_exporter/tasks/install.yml:18]: I see it's also missing the ability to use the latest version lookup that the other roles have, perhaps add that while we are at it?

Comment (SuperQ) [reply roles/blackbox_exporter/tasks/install.yml]: Yea, I also noticed this task is named incorrectly in all the roles.

**Review by gardar — CHANGES_REQUESTED** [review]
Looks good, only thing missing is a molecule scenario for latest version of snmp_exporter

**Comment by SuperQ** [comment]
Whups, I forgot to push that change.

**Pull request merged**